Rajitha/arrayOrderNames.js

```diff
@@ -0,0 +1,49 @@
+/****************************************************************************
+ * A group of friends have decided to start a secret society. The name will 
+ * be the first letter of each of their names, sorted in alphabetical order.
+ * Create a function that takes in an array of names and returns the name of 
+ * the secret society.
+ * ex:societyName(["Adam", "Sarah", "Malcolm"]) ➞ "AMS"
+ *    societyName(["Harry", "Newt", "Luna", "Cho"]) ➞ "CHLN"
+ ***************************************************************************/
+const sortNames = arr => {
+  let firstName;
+  let secondName;
+  let tempName;
+  for (let i = 0; i <= arr.length-2; i++){
```

Comment:
you're mutating the arr.

run this.

```
const arrayTest = ["Banana", "Apple", "Pear", "Carrot"]
console.log(arrayTest)
testSocietyName(arrayTest, "ABCP");
console.log(arrayTest)
```

Reply:
Added new array to avoid mutation and updated the test